src/main/java/com/puppycrawl/tools/checkstyle/checks/annotation/PackageAnnotationCheck.java

```diff
@@ -48,6 +48,21 @@
  * <pre>
  * &lt;module name=&quot;PackageAnnotation&quot;/&gt;
  * </pre>
+ * <p>
+ * Examples of validating files package-info.java and MyClass.java:
+ * </p>
+ *
+ * <p>package-info.java     //ok</p>
```

Comment:
Why do you need those whitespaces between `package-info.java` and `//ok`?

Reply:
Removed the white spaces.